src/prosper/feature.js

```diff
@@ -1 +1,63 @@
-console.log("Prosper")
+/** 
+Activity one (data types)
+--------------------------
+--------------------------
+what data type?
+14 ----------------------- Number
+-80 ---------------------- Number
+8.5 ---------------------- Number
+true --------------------- Boolean
+(5 < 7) ------------------ Boolean
+null --------------------- Undefined
+3.141 -------------------- Number
+"Pi" --------------------- String
+67+18 -------------------- Number 
+**/
+
+
+console.log("Prosper Oyibo");				//log my name to console
+
+convertToBinary = function(x) {
+	//Converts decimal to binary
+	var result = [];						//Initialize result array
+	while (x) {
+		 var remainder = x % 2;				//get modulus of x
+		 result.push(remainder);	 			//store the modulus in the result array
+		 x = (x / 2) | 0;						//get the new value for x
+	} 
+	result.reverse();						// reverse the order of the array
+	return result.join("");					// join the elements of the array and return the result as string
+};
+
+countBs = function(x) {
+	if (typeof(x) === "string" ) {									//check for input type
+		for (var i = 0, len = x.length, count = 0; i < len; i++) {	//initialize variables
+			if (x.charAt(i) === "B") {								//check if the string contains 'B' character
+				count++;											//increment count
+			}
+		}
```

Comment:
Good work with the comments. :+1:  Unfortunately, the code's format makes it difficult for the comments to be read.

You could re-format the code by using consistent spacing so it's easier to read.